src/main/java/org/jitsi/videobridge/Endpoint.java

```diff
@@ -473,13 +473,21 @@ public void send(PacketInfo packetInfo)
             // we can't just reassign a transformed packet back into its
             // proper PacketInfo. Need to change those classes to work with
             // the new packet types
-            VideoRtpPacket[] res = bitrateController.transformRtp(packetInfo);
-            for (VideoRtpPacket videoRtpPacket : res)
+            VideoRtpPacket[] extras = bitrateController.transformRtp(packetInfo);
+            if (extras == null)
+            {
+                logger.warn(
+                    "Dropping a packet which was supposed to be accepted:"
+                        + packet);
+                return;
+            }
+
+            // The original packet was transformed in place.
+            // TODO: should we send this *after* the extras?
```

Comment:
it shouldn't make a difference

Reply:
Doesn't that affect the order in which the packets are sent?

Reply:
yes, and that shouldn't make a difference